pkg/bpf/bpf_linux.go

```diff
@@ -550,6 +551,34 @@ func GetMtime() (uint64, error) {
 	return uint64(unix.TimespecToNsec(ts)), nil
 }
 
+const (
+	timerInfoFilepath = "/proc/timer_list"
+)
+
+// GetJtime returns a close-enough approximation of kernel jiffies
+// that can be used to compare against jiffies BPF helper. We parse
+// it from /proc/timer_list. GetJtime() should be invoked only at
+// mid-low frequencies.
+func GetJtime() (uint64, error) {
+	jiffies := uint64(0)
+	scaler := uint64(8)
+	timers, err := os.Open(timerInfoFilepath)
+	if err != nil {
+		return 0, err
+	}
+	defer timers.Close()
+	scanner := bufio.NewScanner(timers)
+	for scanner.Scan() {
+		tmp := uint64(0)
+		n, _ := fmt.Sscanf(scanner.Text(), "jiffies: %d\n", &tmp)
+		if n == 1 {
+			jiffies = tmp
+			break
+		}
+	}
+	return jiffies >> scaler, nil
```

Comment:
Maybe check `scanner.Err()` here and return the error if any:

```suggestion
	return jiffies >> scaler, scanner.Err()
```
return